src/main/java/com/sureit/admin/attendance/service/EmployeeService.java

```diff
@@ -69,21 +77,26 @@ public boolean deleteEmployeeBasedOnEmpId(int id) {
 		}
 		return status;
 	}
-	
-	public boolean employeeLogin(int id){
-		boolean status = false;
-		try{
+
+	public String createTimeSheetEntry(int id) {

```

Comment:
Modularise this method more. i.e take out validation of existanceof employee in different method and so on..